homeassistant/components/flunearyou/sensor.py

```diff
@@ -156,26 +156,14 @@ def __init__(
     @callback
     def _handle_coordinator_update(self) -> None:
         """Handle updated data from the coordinator."""
-        self.update_from_latest_data()
-        self.async_write_ha_state()
-
-    async def async_added_to_hass(self) -> None:
-        """Register callbacks."""
-        await super().async_added_to_hass()
-        self.update_from_latest_data()
-
-    @callback
-    def update_from_latest_data(self) -> None:
-        """Update the sensor."""
-        raise NotImplementedError
+        self.async_schedule_update_ha_state(force_refresh=True)
```

Comment:
This is not how we should do it. We should let the coordinator entity implement `async_update` where the user can use the entity update service to request a refresh.

I suggest reverting this PR.